[Build Speed] Remove RenderObjectInlines.h from all non-Inlines.h headers

jernoble opened the pull request:
#### d804699e453246e9c6d20523003923130a190697
<pre>
[Build Speed] Remove RenderObjectInlines.h from all non-Inlines.h headers
<a href="https://rdar.apple.com/160223115">rdar://160223115</a>
<a href="https://bugs.webkit.org/show_bug.cgi?id=298613">https://bugs.webkit.org/show_bug.cgi?id=298613</a>

Reviewed by Alex Christensen.

Prior to this patch, RenderObjectInlines.h was included in 8 non-Inlines.h headers. Additionally it
is included 179 times in a WebCore Unified build, and on this machine, each include of that file took the
compiler an average of 1.75s to parse, for a total of 5 CPU minutes of time spent parsing this header.

Three broad categories of definitionions needed from RenderObjectInlines.h were:
- Pulling in node() and related methods
- Pulling in document() and related methods
- Pulling in style() and related methods

Each of these categories of methods have been broken into new headers: RenderObjectNode.h,
RenderObjectDocument.h and RenderObjectStyle.h. Additionally, none of these new headers needs to
include an Inlines.h file itself, as they included only the minimal headers needed for the
definitions included therein.

In many cases, an include of RenderObjectInlines.h could be replaced with one of these new headers.
In others, the include could be removed entirely, and new included added only to implementation files.

In other cases, definitions could be moved out-of-line and into implementation files because they were
either virtual or private.

After this patch, RenderObjectInlines.h was included in 0 non-Inlines.h headers, and is included by
only 1 other Inlines.h header. In a WebCore Unified build, it was only included 110 times, each requiring
only 571ms on average, for a total of 62s of parse time.

* Source/WebCore/Headers.cmake:
* Source/WebCore/SaferCPPExpectations/UncheckedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AXCoreObject.cpp:
* Source/WebCore/accessibility/AccessibilitySpinButton.cpp:
* Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperBase.mm:
* Source/WebCore/accessibility/playstation/AXObjectCachePlayStation.cpp:
* Source/WebCore/accessibility/win/AXObjectCacheWin.cpp:
* Source/WebCore/dom/DocumentMarkerController.cpp:
* Source/WebCore/dom/NodeRenderStyle.h:
* Source/WebCore/editing/Editor.cpp:
* Source/WebCore/editing/InsertLineBreakCommand.cpp:
* Source/WebCore/editing/cocoa/EditingHTMLConverter.mm:
* Source/WebCore/editing/markup.cpp:
* Source/WebCore/html/HTMLMediaElement.h:
(WebCore::HTMLMediaElement::hasRenderer const): Deleted.
* Source/WebCore/html/HTMLVideoElement.h:
* Source/WebCore/inspector/InspectorInstrumentation.cpp:
* Source/WebCore/loader/ImageLoader.cpp:
* Source/WebCore/page/mac/DragControllerMac.mm:
* Source/WebCore/page/mac/EventHandlerMac.mm:
* Source/WebCore/platform/animation/AcceleratedEffect.cpp:
* Source/WebCore/rendering/EventRegion.cpp:
* Source/WebCore/rendering/ImageQualityController.cpp:
(WebCore::ImageQualityController::highQualityRepaintTimerFired):
(WebCore::ImageQualityController::chooseInterpolationQuality):
* Source/WebCore/rendering/ImageQualityController.h:
* Source/WebCore/rendering/RenderBlockInlines.h:
* Source/WebCore/rendering/RenderBoxInlines.h:
* Source/WebCore/rendering/RenderElementInlines.h:
* Source/WebCore/rendering/RenderMedia.h:
(WebCore::HTMLMediaElement::hasRenderer const):
(WebCore::HTMLMediaElement::renderer const):
* Source/WebCore/rendering/RenderObject.cpp:
(WebCore::RenderObject::outlineStyleForRepaint const):
* Source/WebCore/rendering/RenderObject.h:
(WebCore::RenderObject::outlineStyleForRepaint const): Deleted.
* Source/WebCore/rendering/RenderObjectDocument.h:
(WebCore::RenderObject::document const):
(WebCore::RenderObject::protectedDocument const):
(WebCore::RenderObject::isDocumentElementRenderer const):
(WebCore::RenderObject::view const):
(WebCore::RenderObject::renderTreeBeingDestroyed const):
* Source/WebCore/rendering/RenderObjectInlines.h:
(WebCore::RenderObject::isTransformed const):
(WebCore::RenderObject::layoutContext const):
(WebCore::RenderObject::treeScopeForSVGReferences const):
(WebCore::RenderObject::nodeForNonAnonymous const): Deleted.
(WebCore::RenderObject::document const): Deleted.
(WebCore::RenderObject::protectedDocument const): Deleted.
(WebCore::RenderObject::isBody const): Deleted.
(WebCore::RenderObject::isHR const): Deleted.
(WebCore::RenderObject::isPseudoElement const): Deleted.
(WebCore::RenderObject::nonPseudoNode const): Deleted.
(WebCore::RenderObject::writingMode const): Deleted.
(WebCore::RenderObject::node const): Deleted.
(WebCore::RenderObject::protectedNode const): Deleted.
(WebCore::RenderObject::style const): Deleted.
(WebCore::RenderObject::checkedStyle const): Deleted.
(WebCore::RenderObject::isDocumentElementRenderer const): Deleted.
(WebCore::RenderObject::view const): Deleted.
(WebCore::RenderObject::renderTreeBeingDestroyed const): Deleted.
* Source/WebCore/rendering/RenderObjectNode.h:
(WebCore::RenderObject::nodeForNonAnonymous const):
(WebCore::RenderObject::nonPseudoNode const):
(WebCore::RenderObject::node const):
(WebCore::RenderObject::protectedNode const):
(WebCore::RenderObject::isBody const):
(WebCore::RenderObject::isHR const):
(WebCore::RenderObject::isPseudoElement const):
* Source/WebCore/rendering/RenderObjectStyle.h:
(WebCore::RenderObject::writingMode const):
(WebCore::RenderObject::style const):
(WebCore::RenderObject::checkedStyle const):
* Source/WebCore/rendering/RenderTableCellInlines.h:
* Source/WebCore/rendering/RenderTableInlines.h:
* Source/WebCore/rendering/RenderTableSectionInlines.h:
* Source/WebCore/rendering/RenderVideo.h:
(WebCore::HTMLVideoElement::renderer const):
* Source/WebCore/rendering/RenderWidgetInlines.h:
* Source/WebCore/rendering/TextAutoSizing.cpp:
* Source/WebCore/rendering/TextPaintStyle.cpp:
* Source/WebCore/rendering/line/BreakingContext.h:
* Source/WebCore/rendering/line/LineInlineHeaders.h:
* Source/WebCore/rendering/mathml/RenderMathMLToken.cpp:
(WebCore::RenderMathMLToken::setMathVariantGlyphDirty):
* Source/WebCore/rendering/mathml/RenderMathMLToken.h:
(WebCore::RenderMathMLToken::setMathVariantGlyphDirty): Deleted.
* Source/WebCore/rendering/svg/RenderSVGInlineText.h:
* Source/WebCore/rendering/svg/RenderSVGText.cpp:
(WebCore::RenderSVGText::requiresLayer const):
* Source/WebCore/rendering/svg/RenderSVGText.h:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceFilter.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceFilterPrimitive.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceSolidColor.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGRoot.cpp:
* Source/WebCore/style/StyleExtractorConverter.h:
* Source/WebCore/svg/SVGClipPathElement.cpp:
* Source/WebCore/svg/SVGMaskElement.cpp:
* Source/WebCore/testing/InternalSettings.cpp:
* Source/WebKit/Shared/EditingRange.cpp:
* Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundlePage.cpp:
* Source/WebKit/WebProcess/InjectedBundle/DOM/InjectedBundleRangeHandle.cpp:
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm:
* Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm:
* Source/WebKit/WebProcess/WebCoreSupport/ShareableBitmapUtilities.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebEditorClient.cpp:
* Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/DrawingAreaCoordinatedGraphics.cpp:
* Source/WebKit/WebProcess/WebPage/WebPage.cpp:
* Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm:
* Source/WebKit/WebProcess/WebPage/mac/WKAccessibilityWebPageObjectBase.mm:
* Source/WebKit/WebProcess/cocoa/VideoPresentationManager.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebContextMenuClient.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebEditorClient.mm:
* Source/WebKitLegacy/mac/WebView/WebFrame.mm:
* Source/WebKitLegacy/mac/WebView/WebView.mm:

Canonical link: <a href="https://commits.webkit.org/299817@main">https://commits.webkit.org/299817@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a975f8f83dfa6c2d22a3ab2c12abef5aa5c05f63

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/120310 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/40004 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/30655 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/126677 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/72383 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/0871dbba-6b2f-4bf9-ac09-53967b5137a5) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/122186 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/40701 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/48581 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/91371 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/60666 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/902ccbcd-fa92-43cc-b841-de3b8951eeda) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/123262 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/32503 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/107853 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/71924 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/d4b48eb9-d873-4916-ae19-5411888a1f9d) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/31539 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/25957 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/70297 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/101983 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/26142 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/129564 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/47230 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/35834 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/99987 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/47596 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/104036 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/99829 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25339 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/45276 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/23305 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/43863 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/47093 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/52798 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/46561 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/49907 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/48246 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/4431accaa503b3c817df77fc427d9c22b054654d. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50506

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/ec90fb7edcd07904bf8e1f7449a6708db1f4ec71. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50506

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/fbb1255705f2139758288562bcbe3a6cf0287d7f. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50506

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/1091a93c74e96f17dfe708b37d68c02922048815. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50506

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/bf3b64b14cb4f22d86e1cef22b4a96587d5b1266. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50506

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/c25a3e0dfede6574f19955daad8b1c3c40f812da. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50506

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/e3bdbc48e7aab31869246571415c596b05b1da79. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50506

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/adb1dff6a5e99ea98a6b5f32d4556324428672b0. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50506

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/2a3a9cd62cdab6a7fa44ea603fa268f44086acea. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50506

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/a975f8f83dfa6c2d22a3ab2c12abef5aa5c05f63. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50506

jernoble committed:
[Build Speed] Remove RenderObjectInlines.h from all non-Inlines.h headers
rdar://160223115
https://bugs.webkit.org/show_bug.cgi?id=298613

Reviewed by Alex Christensen.

Prior to this patch, RenderObjectInlines.h was included in 8 non-Inlines.h headers. Additionally it
is included 179 times in a WebCore Unified build, and on this machine, each include of that file took the
compiler an average of 1.75s to parse, for a total of 5 CPU minutes of time spent parsing this header.

Three broad categories of definitionions needed from RenderObjectInlines.h were:
- Pulling in node() and related methods
- Pulling in document() and related methods
- Pulling in style() and related methods

Each of these categories of methods have been broken into new headers: RenderObjectNode.h,
RenderObjectDocument.h and RenderObjectStyle.h. Additionally, none of these new headers needs to
include an Inlines.h file itself, as they included only the minimal headers needed for the
definitions included therein.

In many cases, an include of RenderObjectInlines.h could be replaced with one of these new headers.
In others, the include could be removed entirely, and new included added only to implementation files.

In other cases, definitions could be moved out-of-line and into implementation files because they were
either virtual or private.

After this patch, RenderObjectInlines.h was included in 0 non-Inlines.h headers, and is included by
only 1 other Inlines.h header. In a WebCore Unified build, it was only included 110 times, each requiring
only 571ms on average, for a total of 62s of parse time.

* Source/WebCore/Headers.cmake:
* Source/WebCore/SaferCPPExpectations/UncheckedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AXCoreObject.cpp:
* Source/WebCore/accessibility/AccessibilitySpinButton.cpp:
* Source/WebCore/accessibility/mac/WebAccessibilityObjectWrapperBase.mm:
* Source/WebCore/accessibility/playstation/AXObjectCachePlayStation.cpp:
* Source/WebCore/accessibility/win/AXObjectCacheWin.cpp:
* Source/WebCore/dom/DocumentMarkerController.cpp:
* Source/WebCore/dom/NodeRenderStyle.h:
* Source/WebCore/editing/Editor.cpp:
* Source/WebCore/editing/InsertLineBreakCommand.cpp:
* Source/WebCore/editing/cocoa/EditingHTMLConverter.mm:
* Source/WebCore/editing/markup.cpp:
* Source/WebCore/html/HTMLMediaElement.h:
(WebCore::HTMLMediaElement::hasRenderer const): Deleted.
* Source/WebCore/html/HTMLVideoElement.h:
* Source/WebCore/inspector/InspectorInstrumentation.cpp:
* Source/WebCore/loader/ImageLoader.cpp:
* Source/WebCore/page/mac/DragControllerMac.mm:
* Source/WebCore/page/mac/EventHandlerMac.mm:
* Source/WebCore/platform/animation/AcceleratedEffect.cpp:
* Source/WebCore/rendering/EventRegion.cpp:
* Source/WebCore/rendering/ImageQualityController.cpp:
(WebCore::ImageQualityController::highQualityRepaintTimerFired):
(WebCore::ImageQualityController::chooseInterpolationQuality):
* Source/WebCore/rendering/ImageQualityController.h:
* Source/WebCore/rendering/RenderBlockInlines.h:
* Source/WebCore/rendering/RenderBoxInlines.h:
* Source/WebCore/rendering/RenderElementInlines.h:
* Source/WebCore/rendering/RenderMedia.h:
(WebCore::HTMLMediaElement::hasRenderer const):
(WebCore::HTMLMediaElement::renderer const):
* Source/WebCore/rendering/RenderObject.cpp:
(WebCore::RenderObject::outlineStyleForRepaint const):
* Source/WebCore/rendering/RenderObject.h:
(WebCore::RenderObject::outlineStyleForRepaint const): Deleted.
* Source/WebCore/rendering/RenderObjectDocument.h:
(WebCore::RenderObject::document const):
(WebCore::RenderObject::protectedDocument const):
(WebCore::RenderObject::isDocumentElementRenderer const):
(WebCore::RenderObject::view const):
(WebCore::RenderObject::renderTreeBeingDestroyed const):
* Source/WebCore/rendering/RenderObjectInlines.h:
(WebCore::RenderObject::isTransformed const):
(WebCore::RenderObject::layoutContext const):
(WebCore::RenderObject::treeScopeForSVGReferences const):
(WebCore::RenderObject::nodeForNonAnonymous const): Deleted.
(WebCore::RenderObject::document const): Deleted.
(WebCore::RenderObject::protectedDocument const): Deleted.
(WebCore::RenderObject::isBody const): Deleted.
(WebCore::RenderObject::isHR const): Deleted.
(WebCore::RenderObject::isPseudoElement const): Deleted.
(WebCore::RenderObject::nonPseudoNode const): Deleted.
(WebCore::RenderObject::writingMode const): Deleted.
(WebCore::RenderObject::node const): Deleted.
(WebCore::RenderObject::protectedNode const): Deleted.
(WebCore::RenderObject::style const): Deleted.
(WebCore::RenderObject::checkedStyle const): Deleted.
(WebCore::RenderObject::isDocumentElementRenderer const): Deleted.
(WebCore::RenderObject::view const): Deleted.
(WebCore::RenderObject::renderTreeBeingDestroyed const): Deleted.
* Source/WebCore/rendering/RenderObjectNode.h:
(WebCore::RenderObject::nodeForNonAnonymous const):
(WebCore::RenderObject::nonPseudoNode const):
(WebCore::RenderObject::node const):
(WebCore::RenderObject::protectedNode const):
(WebCore::RenderObject::isBody const):
(WebCore::RenderObject::isHR const):
(WebCore::RenderObject::isPseudoElement const):
* Source/WebCore/rendering/RenderObjectStyle.h:
(WebCore::RenderObject::writingMode const):
(WebCore::RenderObject::style const):
(WebCore::RenderObject::checkedStyle const):
* Source/WebCore/rendering/RenderTableCellInlines.h:
* Source/WebCore/rendering/RenderTableInlines.h:
* Source/WebCore/rendering/RenderTableSectionInlines.h:
* Source/WebCore/rendering/RenderVideo.h:
(WebCore::HTMLVideoElement::renderer const):
* Source/WebCore/rendering/RenderWidgetInlines.h:
* Source/WebCore/rendering/TextAutoSizing.cpp:
* Source/WebCore/rendering/TextPaintStyle.cpp:
* Source/WebCore/rendering/line/BreakingContext.h:
* Source/WebCore/rendering/line/LineInlineHeaders.h:
* Source/WebCore/rendering/mathml/RenderMathMLToken.cpp:
(WebCore::RenderMathMLToken::setMathVariantGlyphDirty):
* Source/WebCore/rendering/mathml/RenderMathMLToken.h:
(WebCore::RenderMathMLToken::setMathVariantGlyphDirty): Deleted.
* Source/WebCore/rendering/svg/RenderSVGInlineText.h:
* Source/WebCore/rendering/svg/RenderSVGText.cpp:
(WebCore::RenderSVGText::requiresLayer const):
* Source/WebCore/rendering/svg/RenderSVGText.h:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceFilter.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceFilterPrimitive.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGResourceSolidColor.cpp:
* Source/WebCore/rendering/svg/legacy/LegacyRenderSVGRoot.cpp:
* Source/WebCore/style/StyleExtractorConverter.h:
* Source/WebCore/svg/SVGClipPathElement.cpp:
* Source/WebCore/svg/SVGMaskElement.cpp:
* Source/WebCore/testing/InternalSettings.cpp:
* Source/WebKit/Shared/EditingRange.cpp:
* Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundlePage.cpp:
* Source/WebKit/WebProcess/InjectedBundle/DOM/InjectedBundleRangeHandle.cpp:
* Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.mm:
* Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm:
* Source/WebKit/WebProcess/WebCoreSupport/ShareableBitmapUtilities.cpp:
* Source/WebKit/WebProcess/WebCoreSupport/WebEditorClient.cpp:
* Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/DrawingAreaCoordinatedGraphics.cpp:
* Source/WebKit/WebProcess/WebPage/WebPage.cpp:
* Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm:
* Source/WebKit/WebProcess/WebPage/mac/WKAccessibilityWebPageObjectBase.mm:
* Source/WebKit/WebProcess/cocoa/VideoPresentationManager.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebContextMenuClient.mm:
* Source/WebKitLegacy/mac/WebCoreSupport/WebEditorClient.mm:
* Source/WebKitLegacy/mac/WebView/WebFrame.mm:
* Source/WebKitLegacy/mac/WebView/WebView.mm:

Canonical link: https://commits.webkit.org/299817@main
